src/js/tabs/settingsgateway.js

```diff
@@ -159,7 +161,41 @@ SettingsGatewayTab.prototype.angular = function(module)
               tx.submit();
             });
           }
-          break;  
+          break;
+        case 'globalFreezeFlag':
+          // Need to set flag on account_root only when chosen option is different from current setting
+          if ($scope.currentGlobalFreezeFlagSetting !== $scope.isGlobalFreezeFlagEnabled) {
+            $scope.edit.globalFreezeFlagSaving = true;
+            var tx = network.remote.transaction();
+            !$scope.isGlobalFreezeFlagEnabled ? tx.accountSet(id.account, undefined, Transaction.set_clear_flags.AccountSet.asfGlobalFreeze) : tx.accountSet(id.account, Transaction.set_clear_flags.AccountSet.asfGlobalFreeze);
```

Comment:
These calls are exactly the same, except that one has `undefined` as the second parameter.  Can you comment this, so it's clear that one is adding a flag and the other is removing it?  Alternatively, it looks like you can pass an object with `account, set, and clear`:
```
var transaction = remote.createTransaction('AccountSet', {
  account: MY_ADDRESS,
  set: 'RequireDest',
  clear: 'RequireAuth'
});
```